sidechain/consensus/aura/src/verifier.rs

```diff
@@ -183,6 +189,13 @@ mod tests {
 		) if &m == msg)
 	}
 
+	fn assert_ancestry_mismatch_err<T: Debug>(result: Result<T, ConsensusError>, msg: &str) {
+		assert_matches!(result.unwrap_err(),ConsensusError::BlockAncestryMismatch(
+			_,
+			m,
+		) if &m == msg)
```

Comment:
Is the `unwrap_err()` needed? Does the `assert_matches` not work directly on the result?

Reply:
I think it should work directly with the result. Should I change it? If so,  `assert_bad_sidechain_block_err` fn above should also be changed (as I copy pasted from there :P)